package.json

```diff
@@ -80,11 +83,14 @@
     "@changesets/cli": "^2.17.0",
     "@manypkg/cli": "^0.19.0",
     "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-multi-entry": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.1.3",
+    "@rollup/plugin-typescript": "^8.3.4",
```

Comment:
Since we're using `rollup-plugin-typescript2`, we should be able to uninstall this.

Reply:
Actually, I think we do need this. This plugin is the one which allows us to have a typescript config.

Reply:
Ah I see